mesonbuild/backend/vs2010backend.py

```diff
@@ -966,9 +966,7 @@ def gen_vcxproj(self, target, ofname, guid):
             t_inc_dirs = [self.relpath(self.get_target_private_dir(target),
                                        self.get_target_dir(target))]
             if target.implicit_include_directories:
-                t_inc_dirs += ['.']
-            if target.implicit_include_directories:
-                t_inc_dirs += [proj_to_src_dir]
+                t_inc_dirs += ['.', proj_to_src_dir]
```

Comment:
This change seems unrelated, should be split into a new commit.

Reply:
Split cleanup into own commit.